packages/component/src/Styles/StyleSet/ImageAvatar.ts

```diff
@@ -2,8 +2,9 @@ import CSSTokens from '../CSSTokens';
 
 export default function createImageAvatarStyle() {
   return {
+    aspectRatio: 1,
```

Comment:
Today, we support rectangular avatar. 🙏🏻

https://microsoft.github.io/BotFramework-WebChat/05.custom-components/k.per-message-avatar/comprehensive.html

Type 1, 2, 3, 4 to the bot.

![image](https://github.com/user-attachments/assets/dd1de19f-374b-4ad3-9ebd-fce00841f248)

Reply:
This complicates things :rofl: